data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -13,9 +23,26 @@
       "uniform_name": ""
     }
   ],
-  "associated_entities": [
+  "related_persons": [
```

Comment:
Most likely auto-populated based on associations made at the person level (e.g., when we select a person's birthplace)